src/main/scala/org/broadinstitute/dsde/firecloud/core/ProfileClient.scala

```diff
@@ -258,13 +258,13 @@ class ProfileClientActor(requestContext: RequestContext) extends Actor with Fire
 
   def checkUserInRawls(
     pipeline: WithTransformerConcatenation[HttpRequest, Future[HttpResponse]],
-    requestContext: RequestContext): Future[PerRequestMessage] = {
+    requestContext: RequestContext, userId: String): Future[PerRequestMessage] = {
```

Comment:
By the time the request has made it into orchestration, it has passed through the orchestration proxy so the userInfo object *should* be fully populated.